cibuildwheel/linux.py

```diff
@@ -136,7 +135,6 @@ def run_docker(command, stdin_str=None):
         try:
             run_docker(['create',
                         '--env', 'CIBUILDWHEEL',
-                        '--name', container_name,
```

Comment:
This has nothing to do with the `package_name` variable, so I wouldn't remove it